code/modules/reagents/reagent_containers/food/cans.dm

```diff
@@ -24,6 +28,192 @@
 		return TRUE
 	. = ..()
 
+/obj/item/reagent_containers/food/drinks/cans/update_icon()
+	cut_overlays()
+	var/image/fuseoverlay = image('icons/obj/fuses.dmi', icon_state = "fuse_short")
+	switch(fuselength)
+		if(3 to 5)
```

Comment:
```suggestion
		if(-INFINITY to 5)
```
same here

Reply:
Just caught a bug this causes while testing some changes - this causes cans to spawn with the short overlay on them, so I'm gonna make a fix for that while I'm at it.